Makefile

```diff
@@ -103,9 +103,9 @@ endif
 ifdef ANDROID
 	cd $(INSTALL_DIR)/koreader && \
 		ln -sf ../../$(ANDROID_DIR)/*.lua .
-	@echo "[*] Install afterupdate marker"
-	@echo "# If this file is here, there are no afterupdate scripts in /sdcard/koreader/scripts/afterupdate." > $(INSTALL_DIR)/koreader/afterupdate.marker
 endif
+	@echo "[*] Install update once marker"
+	@echo "# This file indicates, that onetime update once patches have not been applied." > $(INSTALL_DIR)/koreader/update_once.marker
```

Comment:
Please don't change the filename without a good reason. The old one will be still lying around on all android devices where people didn't use this feature. Changing contents it is ok.

Reply:
Good hint. A onetime_upgrade to delete the old `afterupdate_maker` would fix this concern.